AUTHORS

```diff
@@ -0,0 +1,15 @@
+# This is the official list of HTSJDK-Next authors for copyright purposes.
+#
+# The AUTHORS file lists the copyright holders.
+# For example, Broad Institute employees are not listed here, 
+# because Broad Institute holds the copyright.
+
+# This file should be updated whenever a new contributor merges their first commit.
+
+# Names should be added to this file as
+# Organization <webpage>
+# Individual Name <email address or website>
```

Comment:
Should we require a contact person at organizations?  Either the primary committer or their legal/IP person?

Reply:
That's probably a good idea actually.  Some hope of being able to recontact is a good idea.